source/howto/webcomponents.rst

```diff
@@ -0,0 +1,69 @@
+How to Publish your Web Component on the |ODH| Store
+====================================================
+
+If you have developed a :term:`Web Component` that you deem useful for
+the |odh| project or that can be used on top of data provided by the
+|odh|, you can share it and allow other to reuse it, by making it
+freely available on |odh|\'s `Web Components Store
+<https://webcomponents.opendatahub.bz.it/>`_.
+
+The only requirement for all the Web Components offered through the
+Store is that they :strong:`must` be released as an :ref:`Open Source
+Licence <odh-license>`, compatible with those used within the |odh|
+project.
+
+One of three alternatives can be chosen to publish a Web Component on
+the Store: as a :strong:`full open source` project, as a
+:strong:`forked` project, or using what we call the  :strong:`external
+workflow`; the preferred being the first one.
+
+..  Each alternative has its pros and cons, and for each one the terms
+
+#. When using the :strong:`full open source` path, you simply hand in
+   the source code of your Web Component to the |odh| team, no
+   additional effort is required from your side. The code
+   will be placed in the Store's repository, and immediately made
```

Comment:
Would call it "placed in a Git repository", because the Store has no repositories

Reply:
fixed, thanks